components/scream/tests/coupled/homme_physics/CMakeLists.txt

```diff
@@ -16,9 +16,10 @@ set (NEED_LIBS cld_fraction shoc p3 scream_rrtmgp rrtmgp ${NETCDF_C} ${dynLibNam
 
 # Add source files
 set (SRC shoc_cld_p3_rrtmgp.cpp ${SCREAM_BASE_DIR}/src/physics/rrtmgp/tests/rrtmgp_test_utils.cpp)
-
-CreateUnitTest(homme_shoc_cld_p3_rrtmgp "homme_shoc_cld_p3_rrtmgp.cpp" "${NEED_LIBS}" EXE_ARGS "< namelist.nl" LABELS "dynamics;driver;shoc;cld;p3;rrtmgp;physics")
-set_target_properties(homme_shoc_cld_p3_rrtmgp PROPERTIES COMPILE_FLAGS "${YAKL_CXX_FLAGS}")
+if (NOT SCREAM_AUTOTESTER) # TODO: This test should be reinstated once the homme+physics test is fixed.
```

Comment:
Wouldn't this cause nightlies to fail? Can we comment the test altogether then? I get the desire to have the AT passing, but this would cause the nightlies to be constantly red, for a test we know should fail.

Reply:
That's the point, we want the constant reminder from the nightlies that there is a known failed test that still needs to be fixed.  But we don't want the the AT to hold everyone else up.